<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="vegetationPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>427</width>
         <height>293</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="vegetationBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="vegetationTypeLabel">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="vegetationTypeEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="vegetationModelLabel">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="vegetationModelEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="vegetationPositionLabel">
          <property name="text">
           <string>Position</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="vegetationPositionLayout">
          <item>
           <widget class="QSpinBox" name="vegetationPositionXSpin"/>
          </item>
          <item>
           <widget class="QSpinBox" name="vegetationPositionYSpin"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="vegetationRadiusLabel">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="vegetationRadiusSpin"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="vegetationNumberLabel">
          <property name="text">
           <string>Number</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="vegetationNumberSpin"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>427</width>
         <height>293</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
